ansible-tests/validations/neutron-sanity-check.yaml

```diff
@@ -8,6 +8,33 @@
         Neutron's configuration.
       groups:
         - post-deployment
+    config_files:
```

Comment:
The order is important, probably we should add a comment.

`neutron-sanity-check` documentation says:
> values in later files taking precedence